src/pages/task/TestsTableCore.tsx

```diff
@@ -77,17 +81,60 @@ const columns: Array<ColumnProps<TaskTestsData>> = [
   },
   {
     title: "Time",
-    width: "25%",
+    width: "20%",
     dataIndex: "duration",
     key: Categories.Duration,
     sorter: true,
     render: (text: number): string => {
       const ms = text * 1000;
       return msToDuration(Math.trunc(ms));
     }
+  },
+  {
+    title: "Logs",
+    width: "20%",
+    dataIndex: "logs",
+    key: "logs",
+    sorter: false,
+    render: ({
+      htmlDisplayURL,
+      rawDisplayURL
+    }: {
+      htmlDisplayURL: string;
+      rawDisplayURL: string;
+    }): JSX.Element => {
+      return (
+        <>
+          {htmlDisplayURL && (
+            <ButtonWrapper>
+              <Button
+                className="htmlBtn"
+                size="small"
+                target="_blank"
+                variant="default"
+                href={htmlDisplayURL}
+              >
+                HTML
+              </Button>
+            </ButtonWrapper>
+          )}
+          {rawDisplayURL && (
+            <Button
+              className="rawBtn"
```

Comment:
This className is applied so cypress can find this button

Reply:
what about using ids for cypress. we would only use ids for testing which makes it clear it's not for styling

Reply:
Assigning an id to button will result in duplicate id's and I would like to keep all id's unique so we are within the HTML 5.1 spec. It also seems overkill to assign a classname just for testing purposes and I think I need to remove the classname and improve the selector in cypress.

Reply:
Is there a test case where you'd want to find the logs button in a specific row though? If that's the case I think it makes sense to generate an ID that contains the row index

Reply:
> Assigning an id to button will result in duplicate id's and I would like to keep all id's unique so we are within the HTML 5.1 spec. It also seems overkill to assign a classname just for testing purposes and I think I need to remove the classname and improve the selector in cypress.

just append an id to the end of each id